### PR TITLE
Display number of schedules next to each day

### PR DESCRIPTION
**Pull request by stychoi**
Resolves crunchtime issue 17 https://github.com/crunch-time/crunchtime/issues/17

Screenshot 2023-01-04 at 7 20 57 PM

### CODE REVIEW

File: backend/src/main/resources/application.properties [hunk 1]
```diff
@@ -13,7 +13,7 @@ gauchocourses.proxy.server=api.ucsb.edu
 gauchocourses.proxy.port=443
 gauchocourses.proxy.scheme=https
 gauchocourses.proxy.api-key-name=ucsb-api-key
-gauchocourses.proxy.api-key-value=
```
Comment (JasonFreeberg) [inline backend/src/main/resources/application.properties:16]: This shouldn't be necessary, it also causes the build the fail. Are you having trouble connecting to the API without this change?

Comment (stychoi) [reply backend/src/main/resources/application.properties:16]: Yeah looks like it's not needed, I was fiddling around with env variables. But now it's having trouble connecting when I try to load more classes or filter by department

Comment (JasonFreeberg) [reply backend/src/main/resources/application.properties:16]: If you're able, we can do a screenshare during the meeting tonight

Comment (stychoi) [reply backend/src/main/resources/application.properties:16]: Sorry, I wasn't able to make the meeting tonight. Yeah for some reason now it's not connecting unless I paste the env var or the API key on that line.

Comment (JasonFreeberg) [reply backend/src/main/resources/application.properties:16]: What's the name of your env var? If it's "GAUCHOCOURSES_PROXY_API_KEY_VALUE" then Spring _should_ just find and use it

Comment (stychoi) [reply backend/src/main/resources/application.properties:16]: Ah it wasn't set to that. I think it's working now for the moment. Thanks!

**Review by WinbertZhang — APPROVED** [review]
Issue fixed, able to merge to main

**Pull request merged**